bfasst/flows/rand_soc_dumped.py

```diff
@@ -0,0 +1,35 @@
+"""Flow to dump bels from random soc block designs with Isoblaze"""
+import pathlib
+
+from bfasst.flows.flow import FlowNoDesign
+from bfasst.paths import DUMP_TOOL_BUILD_PATH
+from bfasst.tools.design_create.rand_soc import RandSoC
+from bfasst.tools.impl.vivado_impl import VivadoImpl
+from bfasst.tools.synth.vivado_synth_tcl import VivadoSynthFromTcl
+from bfasst.tools.transform.isoblaze_dump import IsoblazeDump
+
+
+class RandSocDumped(FlowNoDesign):
+    """Flow to dump bels from random soc block designs with Isoblaze"""
+
+    def __init__(self, num_designs=1):
+        super().__init__()
+
+        self.rand_soc_tool = RandSoC(self, num_designs=num_designs)
+
+        for design in self.rand_soc_tool.outputs["design_tcl"]:
+            VivadoSynthFromTcl(self, design)
+            VivadoImpl(self, design.parent)
+
+        IsoblazeDump(self, num_designs=num_designs)
```

Comment:
My thinking was just that since the RandSoc tool operates on multiple designs, and we need to perform this "transformation" on each of them, the Dump tool would also operate on the same set of designs. It only adds one or two lines of code to the tool's create_build_snippets method over having a separate instance of the tool for each design created by the RandSoc tool, and probably takes less memory because we don't have to instantiate potentially thousands of copies of the tool. I can change it, if you'd prefer though.

Reply:
Yeah, I think I rather just have it operate on a single design to be consistent with the other tools.

I originally made RandSoC create multiple designs as an exception since it didn't have a "starting design", but I think it might be better to make that one generate a single design and well and move it into the loop.  (but that can be a separate PR)